source/fem-dem/gls_sharp_navier_stokes.cc

```diff
@@ -1193,7 +1222,7 @@ GLSSharpNavierStokesSolver<dim>::output_field_hook(DataOut<dim> &data_out)
       all_shapes.push_back(particle.shape);
     }
   std::shared_ptr<Shape<dim>> combined_shapes =
-    std::make_shared<CompositeShape<dim>>(all_shapes);
+    std::make_shared<CompositeShape<dim>>(all_shapes, Point<dim>(), Point<3>());
```

Comment:
I think it could be a useful option, but that it might become heavy for multiple particles. We could set it as a parameter in the future

Reply:
I agree with @oguevremont it could be added as an option in the future same as the cut cell map output I am working on at the moment. Maybe these options could be under the same parameter like: detailed particles output.